CHANGELOG

```diff
@@ -4,6 +4,10 @@ release of ODP. It incorporates several bug fixes and is otherwise unchanged
 from the base Monarch LTS release
 
 === Bug Fixes (With Numbers)
+==== https://bugs.linaro.org/show_bug.cgi?id=3177[bug 3177]
+Test case for classification enable field in odp_pktin_queue_config
+parameter.
+
```

Comment:
I've been listing bugs in ascending numerical order, so I'd put this at the end of the list rather than the start. I'd also backtick `odp_pktin_queue_config`. But otherwise this looks good.